traffic_ops/traffic_ops_golang/ats/regexrevalidate.go

```diff
@@ -206,44 +180,4 @@ func getMaxDays(tx *sql.Tx) (int64, bool, error) {
 		return 0, false, errors.New("querying max reval duration days: value '" + daysStr + "' is not an integer")
 	}
 	return days, true, nil
-}
-
-func headerComment(tx *sql.Tx, name string) (string, error) {
-	nameVersionStr, err := GetNameVersionString(tx)
-	if err != nil {
-		return "", errors.New("getting name version string: " + err.Error())
-	}
-	return "# DO NOT EDIT - Generated for " + name + " by " + nameVersionStr + " on " + time.Now().Format(HeaderCommentDateFormat) + "\n", nil
-}
-
-func GetNameVersionString(tx *sql.Tx) (string, error) {
-	qry := `
-SELECT
-  p.name,
-  p.value
-FROM
-  parameter p
-WHERE
-  (p.name = 'tm.toolname' OR p.name = 'tm.url') AND p.config_file = 'global'
-`
-	rows, err := tx.Query(qry)
-	if err != nil {
-		return "", errors.New("querying: " + err.Error())
-	}
-	defer rows.Close()
-	toolName := ""
-	url := ""
-	for rows.Next() {
-		name := ""
-		val := ""
-		if err := rows.Scan(&name, &val); err != nil {
-			return "", errors.New("scanning: " + err.Error())
-		}
-		if name == "tm.toolname" {
-			toolName = val
-		} else if name == "tm.url" {
-			url = val
-		}
-	}
-	return toolName + " (" + url + ")", nil
-}
+}
```

Comment:
This looks like it hasn't had `gofmt` run, can you run that?

POSIX files should end with a newline.